cmd/controller-manager/app/options/options.go

```diff
@@ -79,6 +79,29 @@ type Options struct {
 	// It can be set to "0" to disable the metrics serving.
 	// Defaults to ":8080".
 	MetricsBindAddress string
+	// concurrentClusterSyncs is the number of cluster objects that are
+	// allowed to sync concurrently.
+	ConcurrentClusterSyncs int
+	// concurrentClusterAPISyncs is the number of clusterapi controller workers.
+	ConcurrentClusterAPISyncs int
+	// ConcurrentClusterResourceBindingSyncs is the number of clusterresourcebinding objects that are
+	// allowed to sync concurrently.
+	ConcurrentClusterResourceBindingSyncs int
+	// ConcurrentWorkSyncs is the number of work objects that are
+	// allowed to sync concurrently.
+	ConcurrentWorkSyncs int
+	// ConcurrentResourceBindingSyncs is the number of resourcebinding objects that are
+	// allowed to sync concurrently.
+	ConcurrentResourceBindingSyncs int
+	// ConcurrentNamespaceSyncs is the number of name objects that are
+	// allowed to sync concurrently.
+	ConcurrentNamespaceSyncs int
+	// ConcurrentDetectorSyncs is the number of detector workers.
+	// TODO: remove this when PropagationPolicy is separated from the detector
+	ConcurrentDetectorSyncs int
```

Comment:
```suggestion
	// ConcurrentResourceTemplateSyncs is the number of resource templates that are allowed to sync concurrently.
	// Larger number means responsive resource template syncing but more CPU(and network) load.
	ConcurrentResourceTemplateSyncs int
```
We don't need TODO here because:
- 1. we haven't decided to remove PP controller out of the detector( eventhoug I also tend to this approach)
- 2. once we decided to remove PP controller out of the detector, we won't forget to do this.

Reply:
ok